reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -215,9 +217,21 @@ public static Scheduler fromExecutorService(ExecutorService executorService, Str
 	 * between callers. They can however be all {@link #shutdownNow() shut down} together, or replaced by a
 	 * {@link #setFactory(Factory) change in Factory}.
 	 *
+	 * <p>
```

Comment:
IMO the entire javadoc of this method needs to be rearranged. Adding these notes is confusing after having read the above specification. Please mention at the beginning that there are two implementations for the boundedElastic Scheduler - one for reusing a pool of platform threads with idle eviction and another for virtual threads that spawns a new VT for every submitted task while keeping each workers' tasks pipelined to avoid parallel execution. - And that the latter implementation requires JDK21 and a specific property to be set.